content-security-policy/embedded-enforcement/support/echo-required-csp.py

```diff
@@ -1,30 +1,36 @@
 import json
+
+from wptserve.utils import isomorphic_decode
+
 def main(request, response):
     message = {}
 
-    header = request.headers.get("Test-Header-Injection");
-    message['test_header_injection'] = header if header else None
+    header = request.headers.get(b"Test-Header-Injection");
+    message[u'test_header_injection'] = isomorphic_decode(header) if header else None
```

Comment:
Just double checked that `json.dumps` can't deal with bytes as keys, so dealing with strings as much as possible in this test makes sense.